projects/core/src/base-core.module.ts

```diff
@@ -23,6 +24,7 @@ import { StateModule } from './state/state.module';
 
 @NgModule({
   imports: [
+    HttpErrorHandlerModule.forRoot(), // Import this module before any other interceptor to handle HTTP errors efficiently
```

Comment:
[MEDIUM]
Currently in the BaseCoreModule we have 3 error handling modules. What about having a single ErrorHandlingModule moved to the line 27 (with the same code code as before), which internally:
- provides what provided before
- imports HttpErrorHandlerModule
- imports EffectsErrorHandlerModule
?